README.md

```diff
@@ -0,0 +1,65 @@
+# HashiCorp Cloud Platform: HVN for Azure
+
+This Terraform Module provisions a HashiCorp Virtual Network for use with Microsoft Azure.
+
+## Table of Contents
+
+- [HashiCorp Cloud Platform: HVN for Azure](#hashicorp-cloud-platform-hvn-for-azure)
+  - [Table of Contents](#table-of-contents)
+  - [Requirements](#requirements)
+  - [Usage](#usage)
+  - [Author Information](#author-information)
+  - [License](#license)
+
+## Requirements
```

Comment:
It may be helpful to add a requirement around the [API permissions](https://docs.microsoft.com/en-us/cli/azure/ad/app/permission?view=azure-cli-latest) required for the Azure service principal / user you use to run this module. Problematically, if your service principal does not have sufficient AD privileges, it will leave HVN creation in an error state because the peering service principal cannot be created.

Reply:
Added a note on this in the README!